src/py-opentimelineio/opentimelineio/core/_core_utils.py

```diff
@@ -337,7 +337,7 @@ def insert(self, index, item):
                 func = getattr(klass, name)
                 if (
                         isinstance(func, types.FunctionType)
-                        and name not in klass.__abstractmethods__ # noqa and name != "__iter__"
+                        and name not in klass.__abstractmethods__ and not hasattr(sequenceClass, name)  # noqa
```

Comment:
why the noqa?

Reply:
It was because the line was too long, but with your suggestion, I can get rid of it.